overlays/bootstrap.nix

```diff
@@ -691,6 +696,72 @@ in {
 
                 ghc-patches = ghc-patches "9.2.5";
             });
+            ghc941 = final.callPackage ../compiler/ghc (traceWarnOld "9.4" {
+                extra-passthru = { buildGHC = final.buildPackages.haskell-nix.compiler.ghc941; };
+
+                bootPkgs = bootPkgs // {
+                  alex = final.buildPackages.haskell-nix.tool "ghc902" "alex" "3.2.7.1";
+                  happy = final.buildPackages.haskell-nix.tool "ghc902" "happy" "1.20.0";
+                  ghc = final.buildPackages.buildPackages.haskell-nix.compiler.ghc902;
+                };
+                inherit sphinx installDeps;
+
+                useLLVM = !final.stdenv.targetPlatform.isx86 && !final.stdenv.targetPlatform.isAarch64;
```

Comment:
Should we turn this into a `hasNCG: version: arch: ...`? Technically there is a NCG for sparc and power as well. Though I doubt many have systems where that runs on.

Reply:
Perhaps.  I think we can do that in another PR though.